flasksrc/__init__.py

```diff
@@ -50,4 +50,14 @@ def create_app(test_config=None):
     from .survey import ToyNetSurveyById
     api.add_resource(ToyNetSurveyById, '/api/survey/<string:survey_id>')
 
+    from .session import ToyNetSession
+    api.add_resource(ToyNetSession, '/api/toynet/session')
+
+    from .session import ToyNetSessionById
+    api.add_resource(ToyNetSessionById, '/api/toynet/session/<string:toynet_session_id>')
```

Comment:
Hm, should this (an other IDs) be `int`?

Reply:
Yes; confirmed in the DB schema that they are all integers.